include/solvers/stabilization.h

```diff
@@ -30,14 +30,19 @@ using namespace dealii;
  * @param viscosity Kinematic viscosity
  *
  * @param h Cell size. Should be calculated using the diameter of a sphere of equal volume to that of the cell
+ *
+ * @param density Density of the fluid, assumed to be 1 by default.
```

Comment:
I like that this implementation is general! Maybe it would be good to ass a little explanation on why we set the density on 1 by default (eg because we sometimes work with NS equations on rho, and sometimes we need to compute the density, for VOF or rheological properties)?